cypress/.eslintrc.json

```diff
@@ -5,7 +5,7 @@
 
     "rules": {
         // "cypress/no-assigning-return-values": "error",
-        "cypress/no-unnecessary-waiting": 0
+        // "cypress/no-unnecessary-waiting": "error",
```

Comment:
No specific cypress linting rules are used, just the recommended linting rules. Removing this line to ensure any further `no-unnecessary-waiting` errors are not suppressed.

Reply:
Technically no need anymore. I was experimenting with them earlier on. I can just delete the specific `rules` definition entirely now.